exectors/nodejs/displaynames.js

```diff
@@ -0,0 +1,29 @@
+// Tests Intl class DisplayNames
+
+module.exports = {
+
+  testDisplayNames: function (json) {
+    let locale = 'en';  // Default
+    let options;
```

Comment:
Initialization!?

Reply:
This is OK in JS because it sets to "undefined".